addons/binding/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/DigiplexBindingConstants.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
```

Comment:
Please update the license headers. I thing the maven command is `mvn license:update` or similar.